core/network/firewall/rule.go

```diff
@@ -0,0 +1,203 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package firewall
+
+import (
+	"bytes"
+	"fmt"
+	"net"
+	"sort"
+	"strings"
+
+	"github.com/juju/errors"
+	"github.com/juju/utils/set"
+
+	"github.com/juju/juju/core/network"
+)
+
+// AllNetworksIPV4CIDR represents the zero address (quad-zero) CIDR for an IPV4
+// network.
+const AllNetworksIPV4CIDR = "0.0.0.0/0"
```

Comment:
I am using this const in the provider code but kept "0.0.0.0/0" in tests to make sure this does not change. Also, having this will make it easy to figure out what firewall bits need to be changed for IPV6